src/main/java/uk/nhs/tis/trainee/usermanagement/dto/UserAccountDetailsDto.java

```diff
@@ -30,4 +33,6 @@ public class UserAccountDetailsDto {
   String mfaStatus;
   String userStatus;
   List<String> groups;
+  @JsonFormat(shape = Shape.STRING)
```

Comment:
This is new to me, what's the difference in the output JSON?

Reply:
Otherwise it comes out as a datestamp. This just turns it into ISO 8601 format

Reply:
I'm not seeing any difference when I run it locally with and without 🤔

Reply:
Maybe I'm doing this in a clumsy way :/

If I remove that annotation then the `UserAccountResourceTest.shouldGetUserAccountDetails()` test fails with:

```
JSON path "$.accountCreated" expected:<2023-10-18T13:39:43.168048200Z> but was:<1697636383.168048200>
Expected :2023-10-18T13:39:43.168048200Z
Actual   :1697636383.168048200
```

Let me see if I can tweak things to do away with the annotation.

Reply:
Ah, I was just calling it via postman and looking at the result. Didn't try the test, we have `Instant` being returned and tested in other places though.
I'll have a poke too.